node/src/commons/persistent-options.js

```diff
@@ -0,0 +1,75 @@
+const os = require('os');
+const fs = require('fs-extra');
+const path = require('path');
+const { OPTIONS } = require("./constants");
+
+const { API_KEY, API_SECRET, ORGANIZATION_ID, PROJECT_ID, BLUEPRINT_ID, ENVIRONMENT_NAME } = OPTIONS;
+const CONFIG_FILE = path.join(os.homedir(), '.env0', 'config.json');
+
+const envVarToOptionMapper = {
+    ENV0_API_KEY: API_KEY,
+    ENV0_API_SECRET: API_SECRET,
+    ENV0_ORGANIZATION_ID: ORGANIZATION_ID,
+    ENV0_PROJECT_ID: PROJECT_ID,
+    ENV0_BLUEPRINT_ID: BLUEPRINT_ID,
+    ENV0_ENVIRONMENT_NAME: ENVIRONMENT_NAME
+}
+
+const reduceOptions = (config) => {
+    const includedOptions = [ API_KEY, API_SECRET, ORGANIZATION_ID, PROJECT_ID, BLUEPRINT_ID, ENVIRONMENT_NAME ];
```

Comment:
`includedOptions` can be defined outside of the function scope

Reply:
ce3adc5